vite_ruby/vite_ruby.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = Gem::Requirement.new('>= 2.5')
 
-  s.add_dependency 'dry-cli', '~> 0.7.0'
+  s.add_dependency 'dry-cli', '~> 1.0'
```

Comment:
Let's allow `0.7`, as there are no breaking changes.

Reply:
Done. Is it ok to use `< 2`? Or we should prefer `< 1.1`?

Reply:
`< 2` should be fine now that it's a major release. Thanks!